index.html

```diff
@@ -0,0 +1,176 @@
+                    <div class="card__header">
+                        <img src="./assets/svgs/market.svg" alt="Bag">
+                        <h1 class="card__title">
+                            Digital Marketing
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card__header">
+                        <img src="./assets/svgs/card.svg" alt="Bag">
+                        <h1 class="card__title">
+                            Business Growing
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card__header">
+                        <img src="./assets/svgs/users.svg" alt="Bag">
+                        <h1 class="card__title">
+                            Digital Marketing
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card__header">
+                        <img src="./assets/svgs/chart.svg" alt="Bag">
+                        <h1 class="card__title">
+                            National top 50 firms
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card__header">
+                        <img src="./assets/svgs/award.svg" alt="Bag">
+                        <h1 class="card__title">
+                            Web Development
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="video">
+            <div class="video__headline">
+                <h1 class="site-header site-header--medium site-header--center" >
```

Comment:
like at this line `site-header site-header--medium site-header--center` are not meaningfull